src/CONST.ts

```diff
@@ -1522,6 +1522,8 @@ const CONST = {
             ELSEWHERE: 'Elsewhere',
             EXPENSIFY: 'Expensify',
             VBBA: 'ACH',
+            PERSONAL: 'PERSONAL',
+            BUSINESS: 'BUSINESS',
```

Comment:
personal or business are not the `paymentMethodType`. Let's remove them. We pass payAsBusiness:true/false and ELSEWHERE as the `paymentMethodType` for now. The `paymentMethodType` param is related to how the money are moved from the receiver to the invoice sender. `payAsBusiness` is the param based on which we upgrade the receiver and the invoice room to be B2B.
```suggestion

```

Reply:
Hm, okay, get it, removing

Reply:
Let's deal with the individual payments only in this PR. No `payAsBusiness` param and always `ELSEWHERE` as the `paymentMethodType`